src/site/labs/sorting/sorting-basic.rst

```diff
@@ -1,5 +1,5 @@
 *******
-Lab #11
+Sorting
 *******
```

Comment:
maybe, sorting arrays; and the other sorting bags (instead of "more sorting")

Reply:
I changed it to "Sorting Recursively" because I see the labs changing a bit to make use of the bags in the first sorting lab anyways.